packages/core/src/lib/markdown-it/plugins/markdown-it-block-embed/services/VideoServiceBase.js

```diff
@@ -46,6 +46,14 @@ class VideoServiceBase {
     let escapedServiceName = this.env.md.utils.escapeHtml(this.name);
     containerClassNames.push(this.env.options.serviceClassPrefix + escapedServiceName);
 
+    let containerStyles = ["position: relative;", "height: 0;"];
+    // To set a specific style, add a style prop to the getDefaultOptions of any Service
+    if (this.options.style) {
+      containerStyles.push(this.options.style);
+    } else {
+      containerStyles.push(`padding-bottom: ${this.options.height / this.options.width * 100}%`);
+    }
+
     let iframeAttributeList = [];
     iframeAttributeList.push([ "type", "text/html" ]);
     iframeAttributeList.push([ "src", this.getFilteredVideoUrl(videoID) ]);
```

Comment:
One more tiny nit: the `width` / `height` additions to `iframeAttributeList` can probably be removed since you are scaling to `100%/100%`

Reply:
This is used to set the aspect ratio of the embed and then add padding to the bottom according to the aspect ratio. Without it, there would be no space for the embed to be displayed. So I cant remove this else nothing will be shown.

Reply:
![Untitled](https://user-images.githubusercontent.com/3306138/157597372-3a6b9107-7ecb-4580-989c-7de1e40535b1.png)

Note the `padding-bottom` trick works by assuming the child element fills 100% of the container width (achieved by the stylesheet), therefore you can see it as the "height" itself.

Setting a fixed `width` / `height` is unnecessary in the iframe as it is overriden by the stylesheet.

![Untitled](https://user-images.githubusercontent.com/3306138/157597763-afa96cca-7f82-4690-a04e-6f4ef6f68030.png)

Reply:
Oh, sorry! My bad I completely misread your previous post, thought you wanted me to remove the container styles! 

OK anyways it seems like even though I can scale it properly, there is still blurriness, do you have any idea why this might be the case? 

width 320:
![image](https://user-images.githubusercontent.com/19247856/157694988-2b6918a2-75a5-44f1-b292-fc6983e2cea0.png)

Reply:
You'll have to dig into the ppt embed if you are interested; I'm unfortunately not familiar with its implementation details as well. 😐
Though the basic idea would be that the minimum dimensions of the slides decided internally is rather lax (in that the slide almost fills the embed viewport but readability is compromised)